src/js/scrapers/decharlasscraper.js

```diff
@@ -0,0 +1,40 @@
+var DecharlasScraper = function(options) {
+  options = options || {};
+  var url = options.url;
+
+  function canBePublished() {
+    return true;
+  }
+
+  function getTitle(){
+    return document.querySelector('h1.page-title').innerText.trim();
+  }
+
+  function getDescription(){
+    return document.querySelector('div.page_body').innerText;
+  }
+
+  function getDateTime() {
+    return document.querySelector('.meta').innerText;
```

Comment:
scrapper should return the datetime in iso format, that is yyyy-mm-ddTHH:MMZ
it could return anything, but the popup.js is expecting datetime in that format to add it to the time tag directly, I have not tried with other formats.

https://github.com/VLCTechHub/chrome-addon/blob/master/src/js/popup.js#L11
https://github.com/VLCTechHub/chrome-addon/blob/master/src/popup.html#L27